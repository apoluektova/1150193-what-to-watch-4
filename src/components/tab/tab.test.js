import React from "react";
import renderer from "react-test-renderer";
import Tab from "./tab.jsx";

const tabTitles = [`Overview`, `Details`, `Reviews`];

it(`Tab should render correctly`, () => {
  const tree = renderer
     .create(<Tab
       tabTitle={tabTitles[0]}
       onClick={() => {}}
       activeTab={true}
     />)
     .toJSON();

  expect(tree).toMatchSnapshot();
});
