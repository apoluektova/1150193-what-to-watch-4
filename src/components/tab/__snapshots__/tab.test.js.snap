// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab should render correctly 1`] = `
<li
  className="movie-nav__item movie-nav__item--active"
>
  <a
    className="movie-nav__link"
    href="#"
    onClick={[Function]}
  >
    Overview
  </a>
</li>
`;
